current/en-us/7. plugins/2. store.md

```diff
@@ -935,26 +935,58 @@ This is especially useful if you don't want your child-elements to have any know
 In order to do so, you can leverage the higher order function `dispatchify`. What it does is return a wrapped new function which will obtain the store by itself and forward the provided arguments directly to `store.dispatch`.
 
 ```JavaScript Forwarding a dispatchable function as argument to child-elements
-// framework-list.js
-import { inlineView } from 'aurelia-framework';
-import { dispatchify } from 'aurelia-store';
-
-const addFramework = (state, frameworkName) => {
+// actions.js
+export const addFramework = (state, frameworkName) => {
   const newState = Object.assign({}, state);
   newState.frameworks = [...newState.frameworks, frameworkName];
-
+  
   return newState;
 }
 
+// app.html
+<template>
+  <require from="./framework-list" ></require>
+
+  <framework-list></framework-list>
+</template>
+
+// app.js
+import { inject } from 'aurelia-dependency-injection';
+import { Store } from 'aurelia-store';
+
+import * as actions from './actions';
+
+@inject(Store)
+export class App {
+  constructor(store) {
+    this.store = store;
+
+    store.registerAction('Framework::add', actions.addFramework);

```

Comment:
We didn't use so far any namespacing convention or the like. In order to stay consistent with the rest of samples it would be great if you could change this to `Add framework` or `add-framework`. Really just anything else instead of the double colon as this might imply something special which it clearly isn't